MEP_Engine/MEP_Engine.csproj

```diff
@@ -30,23 +30,33 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Analytical_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\Build\Analytical_oM.dll</HintPath>
```

Comment:
Should be set to ProgramData location and not have version/culture/processor architecture